apps/user/stylesheets/alpha.styl

```diff
@@ -22,6 +22,8 @@
   column-count 1
   &__two-columns
     column-count 2
+  &__three-columns
```

Comment:
I don't see the div structure but these are children divs and not modifier classes?